admin_guide/iptables.adoc

```diff
@@ -32,6 +32,13 @@ not impact the operation of {product-title} and the Docker service. Also, change
 will often need to be made on all nodes in the cluster. Use caution, as iptables
 is not designed to have multiple concurrent users and it is very easy to break
 {product-title} and Docker networking.
+
+{product-title} provides one chain which is specifically intended for
```

Comment:
Adding a comma implies (to me) that openshift only provides one chain (and that chain is intended for administrators to use). But it creates many chains, it's just that only one of them has this purposes.